.gitignore

```diff
@@ -59,6 +59,7 @@ coverage.xml
 *.log
 local_settings.py
 db.sqlite3
+test_enumobject
```

Comment:
What's this for?

Reply:
its an outdated thing from the previous way I was mocking django, don't need to worry about it now!